src/main/java/org/eclipse/cargotracker/domain/model/cargo/Leg.java

```diff
@@ -1,28 +1,21 @@
 package org.eclipse.cargotracker.domain.model.cargo;
 
-import java.io.Serializable;
-import java.util.Date;
-
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.Temporal;
-import javax.persistence.TemporalType;
-import javax.validation.constraints.NotNull;
-
 import org.apache.commons.lang3.Validate;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import org.eclipse.cargotracker.domain.model.location.Location;
 import org.eclipse.cargotracker.domain.model.voyage.Voyage;
 
+import javax.persistence.*;
```

Comment:
All the import statements still violate the Google Java Code conventions (https://google.github.io/styleguide/javaguide.html#s3.3-import-statements). Maybe they need to be fixed by applying the formatting rules in IntelliJ?